src/main/java/pl/rejmanbeata/bakery/service/ClientService.java

```diff
@@ -21,20 +22,23 @@ public Client save(ClientEntity clientEntity) {
         return clientMapper.clientEntityToClient(savedClient);
     }
 
-    public List<ClientEntity> getAllClients() {
-        return clientRepository.findAll();
+    public List<Client> getAllClients() {
+        return clientRepository.findAll()
+                .stream()
+                .map(clientMapper::clientEntityToClient)
+                .toList();
     }
 
-    public ClientEntity getClientById(Long id) {
-        return clientRepository.findById(id).orElse(null);
+    public Client getClientById(Long id) {
+        return clientMapper.clientEntityToClient(clientRepository.findById(id).orElse(null));
```

Comment:
I think we can just call `findById(id).get()` (instead of `orElse(null)` since it will have the same result. Or?